webapp/src/app.tsx

```diff
@@ -605,6 +605,42 @@ export class ProjectView
         this.shouldTryDecompile = false;
     }
 
+    async openBlocksAsync() {
```

Comment:
wait what's the difference between this openBlocks and the one above? should there be a comment?